test_graphs.py

```diff
@@ -36,12 +36,19 @@ def pytest_train_model(model_type):
 
     error, true_values, predicted_values = test_trained_model(tmp_file, model_type)
 
+    # Set RMSE and sample error thresholds
+    thresholds = {
```

Comment:
Is it supposed to be a temporary fix?

Reply:
No, I think it's reasonable to have different thresholds per model

Reply:
I would definitely expect to be able to improve GAT though